### ISSUE TITLE
Add an option to "Force Stop" apps instead of "Freeze"

### ISSUE THREAD

**Issue by Nojevah** [FreezeYou/FreezeYou#188]
First, thanks for your great app, it allowed me to freeze/block 2 system apps adb was not able to do before (com.sec.android.sdhms & com.samsung.android.game.gos for the record) and without being rooted: I'm using NO-Root Method (DPC).

My request: Freezing some apps is sometimes extreme for some apps. I'd like an option allowing me to "stop" app, like does SuperFreezZ: it uses an accessibility service to go to the app details and click on " Force Stop".
A bit like in this guide: https://thedroidguy.com/force-quit-app-samsung-1121802

It would not replace the Freeze action, it would complete it, and with that, I'd like a widget allowing me to "force stop" in one click all apps that are in "Force Close" list.

**Comment by confident-hate** [comment]
> 
>I'd like a widget allowing me to "force stop" in one click all apps that are in "Force Close" list.

Have you found anything similar to this? I am currently using SuperfreezZ but it takes too much time to force close apps via accessibility service.

**Comment by Nojevah** [comment]
Honestly, no I don't remember having found something for that. But maybe "Hail" app (which is active and FreezeYou-like) has something like that ? It's the app I'll use when I reinstall my phone (I do this from time to time, ok last time was already 16 months ago).

**Comment by confident-hate** [comment]
Yeah, Hail has a feature that can force stop apps via shizuku. It's fast as hell. Thanks btw

**Comment by Bakr-Ali** [comment]
> Honestly, no I don't remember having found something for that. But maybe "Hail" app (which is active and FreezeYou-like) has something like that ? It's the app I'll use when I reinstall my phone (I do this from time to time, ok last time was already 16 months ago).

Thanks for the "Hail" app recommendation. Exactly what I wanted!
